UnleashedRecomp/main.cpp

```diff
@@ -147,6 +148,29 @@ uint32_t LdrLoadModule(const std::filesystem::path &path)
     return entry;
 }
 
+__attribute__((constructor(101), target("no-avx,no-avx2"), noinline))
+void init()
+{
+#if defined(__x86_64__) || defined(_M_X64)
```

Comment:
The check for MSVC X64 here is nice, but redundant because the __attribute__ tag will fail to compile under MSVC in the first place.